templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -146,6 +146,7 @@ Resources:
           - 'Object Storage Class Changed'
           - 'Object Tags Added'
           - 'Object Tags Deleted'
+          - 'GuardDuty Finding'
```

Comment:
Do we require any docs change for adding this permission by default?

Reply:
docs will be updated saying that guardduty will be enabled by default